docs/api/spinner.md

```diff
@@ -43,6 +43,14 @@ import CSSProps from '@site/static/usage/v7/spinner/theming/css-properties/index
 
 <CSSProps />
 
+## Styling
+
+### Resizing
+
```

Comment:
I wasn't sure what to name/where to put this. I'm also not sure if we want to add this to the docs at all. Feel free to reject if that's appropriate 🙂

Reply:
I lean towards following the structure that buttons has for the [size section](https://ionicframework.com/docs/v6/api/button#size).

Reply:
Hmm, I had looked at that one, but for button, `size` is an attribute, and it seems like most of the top-level items like that are attributes.

Reply:
Good point regarding the attributes.

I also agree that it follows the action sheet more.